cli/lib/util.ts

```diff
@@ -531,14 +535,14 @@ const util = {
     // so for sanity sake we should first trim whitespace characters and remove
     // double quotes around environment strings if the caller is expected to
     // use this environment string as a file path
-    return trim ? dequote(_.trim(result)) : result
+    return trim && result ? dequote(_.trim(result)) : result
```

Comment:
@AtofStryker This check prevents processing of valid string values (like "0") that are falsy but should still be trimmed/dequoted. Would be better to do something like `result != null`

Reply:
good to know. So make sure its not `null` or `undefined` otherwise proceed?

Reply:
added in 67713881e0